src/com/workspace/utill/LogWriter.java

```diff
@@ -1,22 +1,22 @@
 package com.workspace.utill;
-
+import com.workspace.io.*;
 public class LogWriter {
+    private ConsoleWriter consoleWriter = null;
```

Comment:
consoleWriter and fileWriter will be created only once. Right? Can we make it static ?